### PR TITLE
Add treasury.Proposed topic

### LINKED ISSUE #159: Milestone 2 | Push Notifications | Treasury Proposal
> As a user I'd like to receive a push notification when a new treasury proposal is submitted to increase engagement.

- Trigger: When a new treasury proposal is submitted
- UI : No image or icon, only the app logo 
- Copy:
      * Header: "New Treasury Proposal"
      * Body: "A new treasury proposal has been submitted, check it out !"
- Landing screen: Treasury screen
- Timing: (For now) Right after a new treasury proposal has been submitted 
- Should be able to turned it off on the general PN permission screen

### PR DESCRIPTION
**Pull request by ahoseinian**
### Description
Adds relevant push topic (`treasury.Proposed`)

fixes #159

### CODE REVIEW

**Comment by silva-fj** [comment]
Are we considering the network in the deep linking config? We need to change the selected network base on that before navigating to the related screen

**Comment by ahoseinian** [comment]
> Are we considering the network in the deep linking config? We need to change the selected network base on that before navigating to the related screen

now it's added.
I changed the deeplink style to `litentry://api/:network/:redirectTo` for example `litentry://api/kusama/treasury`.

File: src/navigation/AppNavigator.tsx [hunk 1]
```diff
@@ -73,6 +74,10 @@ function DashboardStackNavigator() {
 const Drawer = createDrawerNavigator<DrawerParamList>();
 
 function DrawerNavigator() {
+  useNavigationState((s) => {
```
Comment (silva-fj) [inline src/navigation/AppNavigator.tsx]: This is not used, did you add it for debugging?

**Review by silva-fj — DISMISSED** [review]

File: src/navigation/AppNavigator.tsx (hunk 1 above)
Comment (ahoseinian) [reply src/navigation/AppNavigator.tsx]: oh yes thanks for noticing. will take it out.

**Review by silva-fj — APPROVED** [review]

**Pull request merged**